bluebrain/repo-bluebrain/packages/py-atlas-placement-hints/package.py

```diff
@@ -0,0 +1,34 @@
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyAtlasPlacementHints(PythonPackage):
+    """Library containing command lines and tools to compute placement hints"""
+    homepage = "https://github.com/BlueBrain/atlas-placement-hints"
+    git      = "https://github.com/BlueBrain/atlas-placement-hints.git"
+    pypi     = "atlas-placement-hints/atlas-placement-hints-0.1.1.tar.gz"
+
+    version('develop', branch='main')
+    version('0.1.1', sha256='81d5dcaf5dec607f7a81f730f9e6b9b9567e1e23dabe78d5b397e46564676fb1')
+
+    depends_on('py-atlas-commons@0.1.4:', type=('build', 'run'))
+    depends_on('py-cached-property>=1.5.2', type=('build', 'run'))
```

Comment:
```suggestion
    depends_on('py-cached-property@1.5.2O', type=('build', 'run'))
```
One can interchange `spack`, `module`, and `git`, but not `spack` and `pip` ;)

Reply:
classic; fixed